sherlock/resources/data.json

```diff
@@ -209,6 +209,12 @@
     "urlMain": "https://booth.pm/",
     "username_claimed": "blue"
   },
+  "Backloggery": {
+    "errorType": "status_code",
+    "url": "https://www.backloggery.com/{}",
+    "urlMain": "https://www.backloggery.com/",
+    "username_claimed": "Try4ce"
+  },
```

Comment:
There is a false positve in on this one. If you fix this, this PR will be ready to merge